src/groovy/org/transmartproject/db/util/SimpleScrollableResults.groovy

```diff
@@ -0,0 +1,55 @@
+package org.transmartproject.db.util
+
+import org.hibernate.ScrollableResults
+
+class SimpleScrollableResults<T> implements Closeable, Iterable<T> {
+
+    private final SimpleScrollableResultsIterator scrollableResultsIterator
+
+    SimpleScrollableResults(ScrollableResults scrollableResults) {
+        this.scrollableResultsIterator = new SimpleScrollableResultsIterator()
+        this.scrollableResultsIterator.scrollableResults = scrollableResults
+    }
+
+    @Override
+    Iterator<T> iterator() {
+        scrollableResultsIterator
```

Comment:
If iterator is called twice, you probably don't get what you expect. It would be better to have a guard here throwing `IllegalStateException` if the method is called twice.